src/prism/nickel.js

```diff
@@ -5,13 +5,13 @@
 const nickel = {
     comment: /#.+/,
     string: [
-        { pattern: /m(%+)"(.|\n)*?"\1/, greedy: true},
+        { pattern: /(m|[a-zA-Z][_a-zA-Z0-9-']*-s)(%+)"(.|\n)*?"\2/, greedy: true},
```

Comment:
It's a curiosity, but is our grammar also using an `*` here, meaning that `-s` alone (without a leading character) is a valid symbolic string prefix?

Reply:
Ah, but the Nickel parser doesn't actually accept `-s%""%` as a symbolic string for whatever reason...

Reply:
Sorry, I'm stupid. I didn't see there were two classes of character, so the `*` only applies to the second one, and the first one must appear. All good :+1: